apps/docs/src/app/platform/component-docs/platform-forms/platform-input/platform-input-example/platform-input-example.component.html

```diff
@@ -62,7 +62,16 @@
             placeholder="Field placeholder text"
             hint="Lorem ipsum dolor sit amet, consectetur adipiscing."
         >
-            <fdp-input [name]="'input6'" [id]="'input6'" [type]="'text'"></fdp-input>
+            <fdp-input [name]="'input7'" [id]="'input7'" [type]="'text'"></fdp-input>
         </fdp-form-field>
+        <fdp-form-field
+			#ffl1
+			label="Password Input Field"
+			id="input8"
+			rank="1"
+			placeholder="Field placeholder text"
+        >
+        <fdp-input [name]="'input8'" [id]="'input8'" [type]="'password'" ></fdp-input>
```

Comment:
remove extra bindings present. ex: use name="input8" instead of [name]="'input8'".

Reply:
Seeing this is marked as resolved but the files still show extra bindings?